tests/KdybyTests/Doctrine/config/metadata-empty.neon

```diff
@@ -0,0 +1,6 @@
+parameters:
+	appDir: %wwwDir%/Doctrine/models2 # appDir replacing required - AnnotationDriver::getAllClassNames() problem
```

Comment:
what problem?

Reply:
Problem is line 218 in [AnnotationDriver](https://github.com/doctrine/common/blob/a61bfddd39b887265117806e81cc6ca8da242fe1/lib/Doctrine/Common/Persistence/Mapping/Driver/AnnotationDriver.php#L218). It reload and run again tests/KdybyTests/Doctrine/proxies-sessions-test/run.php, because default value of appDir is tests directory and AnnotationDriver search and load all files in subdirectiories.